metadata.json

```
@@ -128,7 +106,12 @@
     }
   ],
   "requirements": [
-    { "name": "puppet",   "version_requirement": ">=6.0.0 < 8.0.0" }
-  ]
+    {
+      "name": "puppet",
+      "version_requirement": ">=6.0.0 < 8.0.0"
```

we should bump this to now include puppet 8/exclude puppet 6.
```suggestion
      "version_requirement": ">=7.0.0 < 9.0.0"
```

---

changed the version